src/toil/provisioners/abstractProvisioner.py

```diff
@@ -471,28 +471,30 @@ def addSSHRSAKey(self, keyData: str):
             self.sshPublicKeys.append("ssh-rsa " + keyData)
 
 
-        def toCloudConfig(self) -> str:
+        def toIgnitionConfig(self) -> str:
             """
-            Return a CloudConfig configuration describing the desired config.
+            Return a Ignition configuration describing the desired config.
             """
 
             # Define the base config
             config = {
-                'write_files': self.files,
-                'coreos': {
-                    'update': {
-                        'reboot-strategy': 'off'
```

Comment:
Is there no support for configuring Flatcar's update reboot strategy via Ignition?

Reply:
Container Linux Config [still supports this](https://github.com/kinvolk/container-linux-config-transpiler/blob/flatcar-master/doc/configuration.md) as `locksmith.reboot_strategy` being set to `off`, so Ignition must support it. I think https://github.com/kinvolk/container-linux-config-transpiler is just going to turn that into `"locksmith": {"reboot_strategy": "off"}` in the JSON, so you could try adding that? If we really want to be sure that's how Ignition does it, we could build a little CLC file and run the transpiler on it and look at the output.

Reply:
I couldn't find reboot strategy when I was looking at [https://coreos.github.io/ignition/configuration-v3_0/](url) so I thought there may not be support for it in Ignition. But as you mentioned, I think Ignition must support it since CLC still supports it so I'll try adding it in the JSON.

Reply:
There's no explicit option to set `reboot_strategy`. We'd just write the settings to `/etc/coreos/update.conf`. 

```yaml
locksmith:
  reboot_strategy: "off"
```

turns into 

```json
{
  ...
  "storage": {
    "files": [
      {
        "filesystem": "root",
        "path": "/etc/coreos/update.conf",
        "contents": {
          "source": "data:,%0AREBOOT_STRATEGY%3D%22off%22",
          "verification": {}
        },
        "mode": 420
      }
    ]
  }
}
```